### PR TITLE
Story4

### PR DESCRIPTION
**Pull request by Mullivan1**
story 4 completed - pending review notes

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -44,27 +50,72 @@ function checkWord(gameState) {
...
+        gameState.success = true
+    }
     return correctPositions === 5
 }
 
+function resultMessage(result) {
+    let resultArea = document.querySelector('.result')
+    let gameEndMessage = document.querySelector('.game-end-message')
+    let retryButton = document.querySelector('.retry-button')
+    let targetKeyboard = document.querySelector('.keyboard')
```
Comment (ashleycoles) [inline js/index.js]: consts

File: js/index.js [hunk 2]
```diff
@@ -14,21 +15,26 @@ function shuffle(array) {
...
 }
 
 function enterPressed (gameState) {
     if (gameState.attemptedWord.length === 5) {
         const result = checkWord(gameState)
-        alert(result ? 'Correct' : 'Incorrect')
+        gameState.attemptCounter++
+        gameState.attemptedWord = []
+        gameState.characterCounter = 0
+        console.log(gameState)
```
Comment (ashleycoles) [inline js/index.js]: remove logs

File: js/index.js [hunk 3]
```diff
@@ -44,27 +50,72 @@ function checkWord(gameState) {
...
 
+function resultMessage(result) {
+    let resultArea = document.querySelector('.result')
+    let gameEndMessage = document.querySelector('.game-end-message')
+    let retryButton = document.querySelector('.retry-button')
+    let targetKeyboard = document.querySelector('.keyboard')
+
+    if (gameState.attemptCounter === 6 || result === true) {
+        targetKeyboard.classList.toggle('hidden')
+        if (result === true) {
```
Comment (ashleycoles) [inline js/index.js]: No need for the === true

Comment (ashleycoles) [inline js/index.js]: No need for the === true

File: js/index.js [hunk 4]
```diff
@@ -44,27 +50,72 @@ function checkWord(gameState) {
...
     return correctPositions === 5
 }
 
+function resultMessage(result) {
+    let resultArea = document.querySelector('.result')
+    let gameEndMessage = document.querySelector('.game-end-message')
+    let retryButton = document.querySelector('.retry-button')
+    let targetKeyboard = document.querySelector('.keyboard')
+
+    if (gameState.attemptCounter === 6 || result === true) {
```
Comment (ashleycoles) [inline js/index.js]: No need for === true

File: js/index.js [hunk 5]
```diff
@@ -44,27 +50,72 @@ function checkWord(gameState) {
...
+
+
 fetch('words.json')
     .then(response => response.json())
     .then(words => {
         const wordsArr = words['fiveLetterWords']
         const shuffledWords = shuffle(wordsArr)
         gameState.expectedWord = shuffledWords[0].toUpperCase().split("")
-        // console.log(gameState.expectedWord)
+        console.log(gameState.expectedWord)
```
Comment (ashleycoles) [inline js/index.js]: Remove console.log

File: js/index.js [hunk 6]
```diff
@@ -80,15 +131,21 @@ fetch('words.json')
...
-                document.querySelector('.tile' + gameState.characterCounter).textContent = event.key
-                gameState.characterCounter++
-            } else if (event.key == 'Backspace') {
-                deletePressed(gameState)
-            } else if (event.key == 'Enter') {
-                enterPressed(gameState)
+            if (!gameState.success) {
+                if (characterSet.includes(event.key.toLowerCase()) && gameState.attemptedWord.length < 5 ) {
+                    gameState.attemptedWord.push(event.key.toUpperCase())
+                    document.querySelector(`.row${gameState.attemptCounter} .tile${gameState.characterCounter}`).textContent = event.key
```
Comment (ashleycoles) [inline js/index.js]: It seems like this selector is repeated quite a few times in the code:

`.row${gameState.attemptCounter} .tile${gameState.characterCounter}`

Would be worth switching this over to a function that takes in the gameState and returns the query string

File: js/index.js [hunk 7]
```diff
@@ -44,27 +50,72 @@ function checkWord(gameState) {
...
+    gameState.attemptCounter = 0
+    gameState.tryCounter++
+    gameState.expectedWord = e[gameState.tryCounter].toUpperCase().split("")
+    allTiles.forEach(function (tiles) {
+        tiles.textContent = ''
+    })
+    targetKeyboard.classList.remove('hidden')
+    resultArea.classList.add('hidden')
+    retryButton.classList.add('hidden')
+    console.log(gameState.expectedWord)
```
Comment (ashleycoles) [inline js/index.js]: Remove console.log

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Minor changes, nicely done!

File: js/index.js (hunk 1 above)
Comment (bmiller180787) [reply js/index.js]: done

File: js/index.js (hunk 2 above)
Comment (bmiller180787) [reply js/index.js]: done

File: js/index.js (hunk 5 above)
Comment (bmiller180787) [reply js/index.js]: done

File: js/index.js (hunk 7 above)
Comment (bmiller180787) [reply js/index.js]: done

File: js/index.js (hunk 3 above)
Comment (bmiller180787) [reply js/index.js]: done

Comment (bmiller180787) [reply js/index.js]: done

File: js/index.js (hunk 4 above)
Comment (bmiller180787) [reply js/index.js]: done

File: js/index.js (hunk 6 above)
Comment (bmiller180787) [reply js/index.js]: done

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**